### PR TITLE
Feature/add timestamps to csv export

### PR DESCRIPTION
**Pull request by drewpotter**
This is the initial work to add timestamps inside the CSV export for the responses as the column after the `Survey ID` column. In addition to this, I have also improved the CSV filenames to include the timestamp of the download including a `securerandom` 5 character string to make the filenames unique. I have also fixed an AppSignal issue to better use the following code:
```
   rescue => e
     Appsignal.send_error(e)
   end
```
Which is more preferable.

### CODE REVIEW

File: lib/modules/csv_user_exporter.rb [hunk 1]
```diff
@@ -36,6 +37,6 @@ def self.format_user_row(user, default=@@default_na)
   def self.create_filepath
     folder = Rails.root.join("public", "csv_exports")
     FileUtils.mkdir_p(folder)
-    folder.join("csv_user_export_#{DateTime.now.to_i}.csv")
+    folder.join("csv_user_export_#{DateTime.now.to_s}_#{SecureRandom.urlsafe_base64(5)}.csv")
```
Comment (spencerldixon) [inline lib/modules/csv_user_exporter.rb]: Noticed this method is used twice in two different files, may be you could abstract this out and create a CSV parent module which these two can inherit from and benefit from any abstracted helper functions

Comment (drewpotter) [reply lib/modules/csv_user_exporter.rb]: Thanks @spencerldixon I will make that improvement!

**Comment by WKocur** [comment]
@drewpotter  Looks good to me - can I just check what the comment about not interfering with the shiny app is about? Cheers!

**Comment by drewpotter** [comment]
Sure @WKocur basically the CSV format should not change much because there is an app called a shiny app which is not part of this website but which uses the CSV exports as input. So I wanted to put the extra language column at the end of the CSV headers. That is what the last commit is about.

**Comment by WKocur** [comment]
@drewpotter Ah okay - happy to merge then!

**Pull request merged**